package.json

```diff
@@ -5,6 +5,13 @@
   "main": "./lib/index.js",
   "types": "./lib/index.d.ts",
   "type": "commonjs",
+  "browser": {
+    "./lib/index.js": "./lib/index.js",
+    "./lib/hooks/useState":"./lib/hooks/useState.js",
+    "./lib/htm": "./lib/htm/index.js",
+    "./lib/jsx-runtime": "./lib/jsx-runtime/index.js",
+    "./lib/ui": "./lib/ui/index.js"
+  },
```

Comment:
I think we also need to export the /components folder and add an index file inside components?

Reply:
Yup, same for the components. Will push an update 👍